packages/nodes-base/nodes/Airtop/actions/window/close.operation.ts

```diff
@@ -0,0 +1,39 @@
+import type { IExecuteFunctions, INodeExecutionData, INodeProperties } from 'n8n-workflow';
+
+import { validateAirtopApiResponse, validateSessionAndWindowId } from '../../GenericFunctions';
+import { apiRequest } from '../../transport';
+
+export const description: INodeProperties[] = [
+	{
+		displayName: 'Window ID',
+		name: 'windowId',
+		type: 'string',
+		required: true,
+		default: '',
```

Comment:
I believe this should be:
```suggestion
		default: '={{ $json["windowId"] }}',
```
while testing, it defaulted to the following which caused a 400 error since the windowID was missing:
![image](https://github.com/user-attachments/assets/fa14bbef-be2a-4a1e-8e44-67965fef0552)

I would have expected this instead which fixed it:
![image](https://github.com/user-attachments/assets/c784ffc9-42f9-4722-b0b6-9786d7eb9a19)